api/src/geolocate/get.handler.ts

```diff
@@ -21,24 +20,48 @@ async function getGeoDataForLatLong(
   params: SqlParametersList,
   geoid: string,
   table: string,
-): Promise<string | undefined> {
+): Promise<BoundedGeoDatum | undefined> {
+  // TODO(breuch): Consider updating this to all geoids when we support
+  // county, states, etc.
   const executeParams: ExecuteStatementRequest = {
     database: process.env.DATABASE_NAME ?? 'postgres',
     resourceArn: process.env.RESOURCE_ARN ?? '',
     schema: SCHEMA,
     secretArn: process.env.CREDENTIALS_SECRET ?? '',
-    sql: `SELECT ${geoid}
-          FROM ${table}
-          WHERE geom && ST_SetSRID(ST_Point(:long, :lat), 4326)
-          LIMIT 1`,
+    sql: `
+      WITH target_row AS (
+        SELECT ${geoid} AS id, geom AS geom
+        FROM ${table}
+        WHERE geom && ST_SetSRID(ST_Point(:long, :lat), 4326)
+          ORDER BY id ASC
+          LIMIT 1
+      )
+      SELECT
+        STRING_AGG(id, '|') AS id,
```

Comment:
what is the call to STRING_AGG(id, '|') for? can you leave a comment explaining?

Reply:
Done :)